app/routers/profile.py

```diff
@@ -27,20 +31,21 @@
 
 def get_placeholder_user():
     return User(
-        username='new_user',
-        email='my@email.po',
-        password='1a2s3d4f5g6',
-        full_name='My Name',
+        username="new_user",
+        email="my@email.po",
+        password="1a2s3d4f5g6",
+        full_name="My Name",
         language_id=1,
-        telegram_id='',
+        telegram_id="",
     )
 
 
 @router.get("/")
 async def profile(
-        request: Request,
-        session=Depends(get_db),
-        new_user=Depends(get_placeholder_user)):
+    request: Request,
+    session=Depends(get_db),
+    new_user=Depends(get_placeholder_user),
```

Comment:
Use user from Kobi's login system

Reply:
I'm afraid this code is reletedto someone elses feature, im not using the user parameter